### ISSUE TITLE
keras 코드 업로드 완료 / 테스트 결과 

### ISSUE THREAD

**Issue by glee1228** [suimn416/2019.Spring.AI_Leader#41]
parameter& Dataset : 
* Dataset : Sejong University building( in Seoul ) Image Dataset
* input image size : ( 128,128,3 )
* base model : VGG16( pretrained weights : 'imagenet')
* netVLAD clustering num : 7
* train batch size : 32
* training triplet data size : 700 ( 100*3 images per class )
* total epochs  : 10 
* optimizer : SGD(learning rate=0.01, decay=1e-6, momentum=0.9, nesterov=True)
* triplet loss margin : 1e-4

## Triplet Input Set
![581952303 27](https://user-images.githubusercontent.com/26589942/59275784-f3d0ab80-8c97-11e9-8614-06adf0e347ee.png)

## Input Query Image
![581952216 55](https://user-images.githubusercontent.com/26589942/59275796-f8955f80-8c97-11e9-9d56-166f7564b415.png)

## Top 5 Similar Image in reference Dataset
![581952227 80](https://user-images.githubusercontent.com/26589942/59275806-faf7b980-8c97-11e9-9088-824826230e0f.png)

## TSNE Result
![581952195 79](https://user-images.githubusercontent.com/26589942/59275814-fd5a1380-8c97-11e9-9da4-e95bddfcf9df.png)

netVLAD keras version 코드 : https://github.com/glee1228/2019.Spring.AI_Leader/blob/master/netVLAD_triplet_keras.ipynb
